### PR TITLE
[Botskills] Add compatibility for Unix

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
*What is the context of this pull request? Why is it being done?*
Fix case-sensitive issues to make Botskills compatible with Linux.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by Batta32 — APPROVED** [review]
The changes are okay, but I'd like to change the description/title of the PR

e.g. Title: [Botskills] Compatibility with UNIX

**Review by DiegoCardozo94 — APPROVED** [review]
LGMT. Don't forget complete the description of the PR specifically the changes section. 👍

File: tools/botskills/src/utils/sanitizationUtils.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-const regexTrailingBackslash = /.*?(\\)+$/;
+const regexTrailingBackslash = /.*?(\\|\/)+$/;
```
Comment (Batta32) [inline tools/botskills/src/utils/sanitizationUtils.ts:6]: Is this ok?

**Review by Batta32 — COMMENTED** [review]
Is okay that this PR contains tests changes?

File: tools/botskills/src/utils/sanitizationUtils.ts (hunk 1 above)
Comment (VictorGrycuk) [reply tools/botskills/src/utils/sanitizationUtils.ts:6]: It is not part of the tests, they remove the trailing `\` or `/` from file paths.
It should be used for arguments that are paths entered by the user

**Pull request closed without merging**